### ISSUE TITLE
metaWRAP and Illumina nextSeq data

### ISSUE THREAD

**Issue by jarrodscott** [bxlab/metaWRAP#51]
Hi!

Can `metawrap read_qc` handle NextSeq data since there are 4 forward and 4 reverse fastq files?

The answer from the help menu seems to be no but thought I would ask just the same. 

Thanks.

**Comment by ursky** [comment]
Yes, but you would have to run Read_qc 4 times - once for each read pair. See the tutorial for an example of how to do this efficiently.

**Comment by jarrodscott** [comment]
Perfect! Thanks for the blazing fast reply.

**Comment by ursky** [comment]
Reopening for visibility.

**Comment by jarrodscott** [comment]
oops! sorry.

**Comment by jarrodscott** [comment]
Of course. I have 8 fastq files per sample. Like so:

BM1_1909_S61_L001_1.fastq
BM1_1909_S61_L001_2.fastq
BM1_1909_S61_L002_1.fastq
BM1_1909_S61_L002_2.fastq
BM1_1909_S61_L003_1.fastq
BM1_1909_S61_L003_2.fastq
BM1_1909_S61_L004_1.fastq
BM1_1909_S61_L004_2.fastq

And I run the following command:
`metawrap kraken -o TEST -t 24 -s 1000000 CLEAN_READS/BM1_1909_S61*`

from this I get four each of `.krona` and `.kraken` files, plus a `kronagram.html` with four separate plots. The job finishes fine but I am getting this error message as well:

`syntax error: invalid arithmetic operator (error token is ".krak`

FYI, I am working off an old build (from October) and am updating metaWRAP now so I will try again when it is finished installing. 
[job_03_kraken_test.log](https://github.com/bxlab/metaWRAP/files/2750207/job_03_kraken_test.log)

**Comment by ursky** [comment]
This is the intended behavior of the module. You provided four samples - BM1_1909_S61_L001, BM1_1909_S61_L002, BM1_1909_S61_L003, BM1_1909_S61_L004 - and metaWRAP would not know that they are actually the same sample. You will need to merge the reads into single files for it to treat it as such. This goes for every other module as well. For example: 
```
cat BM1_1909_S61*_1.fastq > BM1_1909_S61_1.fastq
cat BM1_1909_S61*_2.fastq > BM1_1909_S61_2.fastq
 metawrap kraken -o TEST -t 24 -s 1000000 CLEAN_READS/BM1_1909_S61_1.fastq CLEAN_READS/BM1_1909_S61_2.fastq
```

**Comment by jarrodscott** [comment]
perfect! Thanks for clarifying. Easy enough. And thanks also for the great tool.
